.travis.yml

```diff
@@ -4,32 +4,24 @@ branches:
 
 matrix:
   include:
-    - os: linux
-      language: generic
-      dist: xenial
-      sudo: required
-      env: SWIFT_VERSION="5.1"
     - os: linux
       language: generic
       dist: xenial
       sudo: required
       env: 
         - SWIFT_VERSION="5.1"
-        - COMBINEX_EXPERIMENTAL_OBSERVABLE_OBJECT = YES
-    - os: osx
-      language: swift
-      osx_image: xcode11.1
-      env: SWIFT_PACKAGE_USE_COMBINEX="YES"
+        - CX_EXPERIMENTAL_OBSERVABLE_OBJECT="YES"
```

Comment:
So we don't test with experimental features disabled? even if most users use CombineX this way.

We could accidentally use experimental components in normal code.

Reply:
@ddddxxx Wow, I didn't think of this. I will bring it back.